### PR TITLE
Blog: Automation API Cloud Tooling

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/automation-api-supercharged-cloud-tooling/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Automation API: Supercharged Cloud Tooling"
+date: 2021-01-14
+meta_desc: "Pulumi's Automation API enables you to build cloud toos to increase developer productivity."
```
Comment (ericrudder) [inline content/blog/automation-api-supercharged-cloud-tooling/index.md:4]: perhaps tools?

File: content/blog/automation-api-supercharged-cloud-tooling/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Automate All the Things
+
+Cloud engineering will automate many tasks that are currently performed by operators. Using infrastructure as code and SDKs such as Automation API, we can build self-service tools that improve the inner dev loop for application developers and let operators perform higher-level tasks such as maintenance and infrastructure monitoring. This simple example demonstrates the effective use of infrastructure as code by importing other software libraries to build a specialized tool that is not possible with DSLs or YAML.
+
+Automation API is currently in preview and will be part of the Pulumi 3.0 GA release, including additional language support. To get started with Automation API, try out the [examples](https://github.com/pulumi/automation-api-examples) and documentation for [node.js]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/x/automation" >}}) and [go](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v2/go/x/auto). If you are looking for a more complex example, check out [ploy](https://github.com/jaxxstorm/ploy), a CLI tool that builds  Docker images and deploys them on Kubernetes.
```
Comment (komalali) [inline content/blog/automation-api-supercharged-cloud-tooling/index.md:58]:
```suggestion
Automation API is currently in preview and will be part of the Pulumi 3.0 GA release, including additional language support. To get started with Automation API, try out the [examples](https://github.com/pulumi/automation-api-examples) and documentation for [node.js]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/x/automation" >}}) and [go](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v2/go/x/auto). If you are looking for a more complex example, check out [ploy](https://github.com/jaxxstorm/ploy), a CLI tool that builds Docker images and deploys them on Kubernetes.
```

File: content/blog/automation-api-supercharged-cloud-tooling/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Infrastructure as code enables you to build tools and environments to automate routine tasks, letting cloud engineers concentrate on efficiency and resilience. In this article, we'll take a look at how Pulumi's Automation API lets you build custom ops tooling that improves your workflow.
+
+<!--more-->
+
+## What is the Automation API?
+
+Automation API is a programming interface overlaid on the Pulumi engine for deploying infrastructure. Instead of using the Pulumi CLI to deploy infrastructure, you can directly call the engine to run code from your chosen language. With the Automation API, your Pulumi programs and stacks are strongly-typed and composable infrastructure building blocks. You can fully embed Pulumi inside your software projects, which means your code can power a wide range of cloud automation projects without the Pulumi CLI and a reduced human-in-the-loop footprint.
+
+{{% notes type="info" %}}
+This alpha release of Automation API currently supports Node.js and go. Python and .NET (C#) will be released soon.
```
Comment (komalali) [inline content/blog/automation-api-supercharged-cloud-tooling/index.md:24]: Python was released today as part of 2.17.1

```suggestion
This alpha release of Automation API currently supports Node.js, Go and Python. .NET (C#) will be released soon.
```

**Review by komalali — APPROVED** [review]
Couple of suggestions since python was released today but otherwise LGTM!

**Comment by EvanBoyle** [comment]
Looks great! Sorry I didn't get a chance to review this last week.

**Pull request merged**